### PR TITLE
Addressing out of date docs

### PR DESCRIPTION
**Pull request by rschwabco**

### CODE REVIEW

File: docs/getting-started-acmecorp/create-policy.mdx [hunk 1]
```diff
@@ -16,9 +16,13 @@ To get started, we'll create a policy repository and a single policy instance th
 
 ## Add a policy repository
 
-After creating your organization, you'll be redirected to the Registry tab. Click on the "Create a new repository" button.
+After creating your organization, navigate to the Images tab.
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/create-policy.mdx:19]: Is it better to have the user just create a policy from a public image? 

Then, during the "Modify the policy" topic, you could have the user create a new image from a template?

File: docs/getting-started-acmecorp/create-policy.mdx [hunk 2]
```diff
@@ -86,6 +92,10 @@ First, click on the "Policies" tab and click the "Create an instance" button:
 
 <img src="/getting-started-acmecorp/create-instance-button.png" width="170" />
 
+Next, you'll be asked to select between creating the policy from a sample image or from your own policy image - choose "...from your own policy image".
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/create-policy.mdx:95]: Isn't there a missing step here where you ask the user to navigate to the Policies tab?

File: docs/getting-started-acmecorp/set-up-users.mdx [hunk 3]
```diff
@@ -31,19 +31,19 @@ and click “Add a connection”
...
 />
 
-Next, select the `acmecorp` provider:
+Next, select the "Identity Providers" connection type and select the the `Demo Acmecorp IDP` provider:
 
-![add-acmecorp-connections](/getting-started-acmecorp/add-acmecorp-connections.png)
+![add-acmecorp-connection](/add-acmecorp-connection.png)
 
-Name the provider "acmecorp" and give it a description.
+Name the provider "acmecorp" and use the display name "Acmecorp."
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/set-up-users.mdx]: name the connection "acmecorp"

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: docs/getting-started-acmecorp/create-policy.mdx (hunk 1 above)
Comment (rschwabco) [reply docs/getting-started-acmecorp/create-policy.mdx:19]: I think this should walk them through the process of creating the policy as code flow, and have the new "Getting Started" guide (which will be short) use a policy from a public image.

File: docs/getting-started-acmecorp/create-policy.mdx (hunk 2 above)
Comment (rschwabco) [reply docs/getting-started-acmecorp/create-policy.mdx:95]: Line 91 says: 
```
First, click on the "Policies" tab and click the "Create an instance" button:
```

**Review by ogazitt — APPROVED** [review]

**Pull request merged**